content/guides/user-guides/multi-cloud/storage-service-plan/_index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Storage Service Plan
+description: "User guide for Storage Service Plan(SSP) Management."
+weight: 10
+disableToc: false
+tags: ["user guide", "storage service plan", "multi-cloud"]
+---
+
+## Introduction to Storage Service Plan(SSP) management
+SODA is aimed at addressing the storage integration challenges of both the Cloud Native environment and traditional IT environment. SODA multi-cloud allows service providers(admin) or users to manage distributed cloud and on-premise environments from a single platform.
+
+
+SSP management allows service providers to abstract actual cloud or on-premise environments(backends) from their users.
```

Comment:
on-premise environments - this is not yet supported

Reply:
Ceph is a on-prem storage